generator/Makefile

```diff
@@ -13,16 +13,18 @@
 
 MIBDIR := mibs
 
-APC_URL        := 'https://download.schneider-electric.com/files?p_File_Name=powernet426.mib'
-ARISTA_URL     := https://www.arista.com/assets/data/docs/MIBS
-CISCO_URL      := 'ftp://ftp.cisco.com/pub/mibs/v2/v2.tar.gz'
-KEEPALIVED_URL := 'https://raw.githubusercontent.com/acassen/keepalived/master/doc/KEEPALIVED-MIB.txt'
-MIKROTIK_URL   := 'http://download2.mikrotik.com/Mikrotik.mib'
-PALOALTO_URL   := 'https://www.paloaltonetworks.com/content/dam/pan/en_US/assets/zip/technical-documentation/snmp-mib-modules/PAN-MIB-MODULES-7.0.zip'
-SERVERTECH_URL := 'ftp://ftp.servertech.com/Pub/SNMP/sentry3/Sentry3.mib'
-SYNOLOGY_URL   := 'https://global.download.synology.com/download/Document/MIBGuide/Synology_MIB_File.zip'
-UBNT_DL_URL    := http://dl.ubnt-ut.com/snmp
-UBNT_AIROS_URL := https://dl.ubnt.com/firmwares/airos-ubnt-mib/ubnt-mib.zip
+APC_URL          := 'https://download.schneider-electric.com/files?p_File_Name=powernet426.mib'
+ARISTA_URL       := https://www.arista.com/assets/data/docs/MIBS
+CISCO_URL        := 'ftp://ftp.cisco.com/pub/mibs/v2/v2.tar.gz'
+KEEPALIVED_URL   := 'https://raw.githubusercontent.com/acassen/keepalived/master/doc/KEEPALIVED-MIB.txt'
+MIKROTIK_URL     := 'http://download2.mikrotik.com/Mikrotik.mib'
+PALOALTO_URL     := 'https://www.paloaltonetworks.com/content/dam/pan/en_US/assets/zip/technical-documentation/snmp-mib-modules/PAN-MIB-MODULES-7.0.zip'
+SERVERTECH_URL   := 'ftp://ftp.servertech.com/Pub/SNMP/sentry3/Sentry3.mib'
+SYNOLOGY_URL     := 'https://global.download.synology.com/download/Document/MIBGuide/Synology_MIB_File.zip'
+UBNT_DL_URL      := http://dl.ubnt-ut.com/snmp
+UBNT_AIROS_URL   := https://dl.ubnt.com/firmwares/airos-ubnt-mib/ubnt-mib.zip
+IANA_PRINTER_URL := https://www.iana.org/assignments/ianaprinter-mib/ianaprinter-mib
+PRINTER_URL      := https://raw.githubusercontent.com/librenms/librenms/master/mibs/Printer-MIB
```

Comment:
Do have a more official source for this?

Reply:
Unfortunately not that I could find. It's not shipped with Net-SNMP.

Reply:
I suppose it's also available at https://www.activexperts.com/admin/mib/RFC/Printer-MIB/ for example, but idk if that qualifies as "more official".

Reply:
librenms is not an official source as they just collect MIBs from everywhere. I want to keep this to official vendor locations, which I think is ftp://ftp.pwg.org/pub/pwg/pmp/mibs/prtmib15.mib in this case.

Reply:
Nice find, thank you! I'll update the location.

Reply:
Mmm. Looks like pinter15.mib is older than the Printer-MIB-v2, 2003 vs 2004. 1.5 looks like a work in progress during RFC 3805.

Reply:
Ah: https://ftp.pwg.org/pub/pwg/pmp/mibs/rfc3805b.mib looks like what we want.

Reply:
Please also add https://www.iana.org/assignments/ianaprinter-mib/ianaprinter-mib